notification/src/main/java/com/iluwatar/notification/App.java

```diff
@@ -0,0 +1,30 @@
+package com.iluwatar.notification;
+
+/**
+* With the DAO pattern, we can use various method calls to
+* retrieve/add/delete/update data without directly interacting
+* with the data source. The below example demonstrates basic CRUD
+* operations: select, add, update, and delete.
+*/
```

Comment:
The text seems to be copied from another pattern. Please update it to introduce the notification pattern briefly and describe how the example code implements it.